### ISSUE TITLE
Disable the "What's New" window

### ISSUE THREAD

**Issue by johnbillion** [alefragnani/vscode-project-manager#238]
As much as I enjoy using this extension, I don't want to be interrupted by a "What's new" window each time the plugin gets updated. I also use your `Bookmarks` extension, and being shown two "What's new" windows in quick succession isn't a great user experience.

May I recommend that the notifications system in Code is used instead to deliver these update notifications?

**Comment by alefragnani** [comment]
Hi @johnbillion ,

I'm glad to know you use both of my most downloaded extensions 😬 .

Thank you for your suggestion to use the _notification system_, but I'll keep the current approach. Don't worry, the "What's New" window will appear only on new releases. This is not a nag window. 

You received two in succession because I'm applying the same feature for all of my extensions. There are 6 more to go...

**Comment by tadly** [comment]
Agreed.

If people don't read it, immediately close it and even go as far as to search for an option to disable this functionality, then there's really no reason to **force** it.

Please reconsider.

**Comment by xididri** [comment]
I have multiple VS Code windows and once WSL disconnect the "what's new" window popup and needs to be closed on all of them, when I happen to disconnect/reconnect several times it is getting really annoying.

I do not care what is new in Project Manager and I think most people don't.

Please consider this option 🙏

**Comment by kankaristo** [comment]
To hide the "What's new" window, you can modify VSCode `globalStorage`, and say that installed plugin version is e.g. 999.0.0, so that the plugin doesn't know when it's upgraded. I might be dumb posting this on here, since the extension author might change the logic to work around this, but anyway...

I use a Bash script to open VSCode (to open multiple windows), so I added this to that script (use at your own risk):
```bash
#
# Hide the "What's new" page for some VSCode extensions.
#
function hideWhatsNew {
    globalStoragePath="$HOME/.config/Code/User/globalStorage/state.vscdb"
    if [ ! -f "$globalStoragePath" ]; then
        >&2 echo "ERROR: Cannot find VSCode globalStorage SQLite file: $globalStoragePath"
        return 64
    fi
    if ! command -v sqlite3 &> /dev/null; then
        >&2 echo "ERROR: Command not found (not installed?): sqlite3"
        return 65
    fi
    
    # alefragnani.Bookmarks
    sqlite3 "$globalStoragePath" \
        "UPDATE ItemTable SET value = json_set(value, '$.\"Bookmarks.version\"', '999.0.0') WHERE key = 'alefragnani.Bookmarks';"
    # alefragnani.numbered-bookmarks
    sqlite3 "$globalStoragePath" \
        "UPDATE ItemTable SET value = json_set(value, '$.\"numbered-bookmarks.version\"', '999.0.0') WHERE key= 'alefragnani.numbered-bookmarks';"
    # alefragnani.project-manager
    sqlite3 "$globalStoragePath" \
        "UPDATE ItemTable SET value = json_set(value, '$.\"project-manager.version\"', '999.0.0') WHERE key= 'alefragnani.project-manager';"
}

# Call the above function before starting VSCode:
hideWhatsNew
```

**Comment by UnstoppableMango** [comment]
I use this extension on Nix and due to the immutability of the nix store, VSCode appears freshly installed nearly every time it opens. That means I also see the Project Manager "What's New" tab nearly every time I open VSCode. It would be nice to have an option to simply disable it!
